src/test/java/com/ft/methodeimagemodelmapper/resources/MethodeImageModelResourceCT.java

```diff
@@ -0,0 +1,177 @@
+package com.ft.methodeimagemodelmapper.resources;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.ft.messagequeueproducer.MessageProducer;
+import com.ft.methodeimagemodelmapper.MethodeImageModelMapperApplication;
+import com.ft.methodeimagemodelmapper.configuration.MethodeImageModelMapperConfiguration;
+import com.ft.methodeimagemodelmapper.configuration.ProducerConfiguration;
+import com.sun.jersey.api.client.Client;
+import com.sun.jersey.api.client.ClientResponse;
+import io.dropwizard.setup.Environment;
+import io.dropwizard.testing.junit.DropwizardAppRule;
+import org.apache.commons.lang.StringUtils;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+
+import javax.ws.rs.core.MediaType;
+import java.net.URI;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+import static com.ft.api.util.transactionid.TransactionIdUtils.TRANSACTION_ID_HEADER;
+import static javax.ws.rs.core.HttpHeaders.CONTENT_TYPE;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyZeroInteractions;
+
+public class MethodeImageModelResourceCT {
```

Comment:
The name of this file means by default it is not executed by Maven. You need to either name it `...Test`, or change the POM file to include it.